service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/cmo/ReviewCMOController.java

```diff
@@ -118,9 +119,13 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
                 data.put("sealedCMOs", sealedCMOs);
 
                 if (featureToggleService.isNewCaseStateModelEnabled()
-                    && caseData.getReviewCMODecision().hasReviewOutcomeOf(SEND_TO_ALL_PARTIES)
-                    && isNextHearingOfType(caseData, cmo.getId(), ISSUE_RESOLUTION)) {
-                    data.put("state", State.ISSUE_RESOLUTION.getValue());
+                    && caseData.getReviewCMODecision().hasReviewOutcomeOf(SEND_TO_ALL_PARTIES)) {
+
+                    if (isNextHearingOfType(caseData, cmo.getId(), ISSUE_RESOLUTION)) {
+                        data.put("state", State.ISSUE_RESOLUTION.getValue());
```

Comment:
Again could remove State. and use ISSUE_RESOLUTION if you want - minor

Reply:
Yeah I purposely left this extended to State.ISSUE_RESOLUTION to differ between hearing type of same name - line 124. I think it was easier to read at a glance. I then done the same thing with 'State.FINAL_HEARING' just to follow this pattern.

Thoughts?

Reply:
Oh yes this makes sense for me.